### PR TITLE
i18n(ru): Primary pages were translated to russian

### PR DESCRIPTION
**Pull request by Lootjs**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

I would like to share my translation of some documentation pages into Russian.
- Also, I've added Russian to the language selection block.
- In addition, I've corrected some inaccurate translations into Russian in the UI.

image

Could someone add the hacktoberfest-accepted label? 👀

p.s here is my previous [PR to Astro Docs](https://github.com/withastro/docs/pull/1735)

### CODE REVIEW

**Comment by Lootjs** [comment]
Is it necessary to add changesets? 🤔

**Review by delucis — COMMENTED** [review]
Thank you @Lootjs! I can see one small change needed here, otherwise looks great 🙌

File: docs/src/content/docs/ru/index.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+template: splash
+editUrl: false
+lastUpdated: false
+banner:
+  content: |
+    <span class="ph-banner">
+      <span class="ph-houston"><img src="/ph-houston.svg" alt="" width="72" height="72"/></span>
+      <span class="ph-text">
+        <strong>Welcome Product Hunters&nbsp;👋</strong>
+        <a href="https://www.producthunt.com/posts/starlight-by-astro">We are live right now!</a>
```
[the remark this answers is not in the document]
Comment (Lootjs) [reply docs/src/content/docs/ru/index.mdx:16]: 👌

[the remark this answers is not in the document]
Comment (Lootjs) [reply docs/src/content/docs/ru/index.mdx]: The revisions are finished 😎

**Review by delucis — APPROVED** [review]
Thank you! Ready to launch 🚀

File: docs/src/content/docs/ru/getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+[откройте шаблон на StackBlitz](https://stackblitz.com/github/withastro/starlight/tree/main/examples/basics).
+:::
+
+## Создание контента с Starlight
+
+Starlight готов к созданию нового контента или к интеграции уже существующих файлов!
+
+### Форматы файлов
+
+Starlight поддерживает создание контента в форматах Markdown и MDX. (Вы можете добавить поддержку Markdoc, установив экспериментальную [интеграцию Astro Markdoc](https://docs.astro.build/en/guides/integrations-guide/markdoc/).)
```
Comment (delucis) [inline docs/src/content/docs/ru/getting-started.mdx:54]:
```suggestion
Starlight поддерживает создание контента в форматах Markdown и MDX. (Вы можете добавить поддержку Markdoc, установив экспериментальную [интеграцию Astro Markdoc](https://docs.astro.build/ru/guides/integrations-guide/markdoc/).)
```

File: docs/src/content/docs/ru/getting-started.mdx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+## Устранение проблем с Starlight
+
+Информация о [конфигурации проекта Starlight](/ru/reference/configuration/)
+и [конфигурации метаданных отдельной страницы](/ru/reference/frontmatter/) доступна в разделе Справочник этого сайта.
+Используйте эти страницы, чтобы убедиться, что ваш сайт на Starlight настроен и работает корректно.
+
+Смотрите пополняющийся список руководств в боковой панели для помощи в добавлении контента и настройке вашего сайта на Starlight.
+
+Если вы не можете найти ответ в этой документации, пожалуйста, посетите [полную документацию Astro](https://docs.astro.build).
```
Comment (delucis) [inline docs/src/content/docs/ru/getting-started.mdx:133]:
```suggestion
Если вы не можете найти ответ в этой документации, пожалуйста, посетите [полную документацию Astro](https://docs.astro.build/ru/).
```

File: docs/src/content/docs/ru/guides/authoring-content.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+Вы можете выделить `код` обратными кавычками.
+
+```md
+Вы можете выделить `код` обратными кавычками.
+```
+
+## Изображения
+
+Изображения в Starlight используют [встроенную оптимизацию ресурсов в Astro](https://docs.astro.build/en/guides/assets/).
```
Comment (delucis) [inline docs/src/content/docs/ru/guides/authoring-content.md:34]:
```suggestion
Изображения в Starlight используют [встроенную оптимизацию ресурсов в Astro](https://docs.astro.build/ru/guides/assets/).
```

File: docs/src/content/docs/ru/guides/authoring-content.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+
+:::tip[Знали ли вы?]
+Astro позволяет создавать быстрые сайты с помощью ["архитектуры островов"](https://docs.astro.build/en/concepts/islands/)
+:::
+
+```md
+:::tip[Знали ли вы?]
+Astro позволяет создавать быстрые сайты с помощью ["архитектуры островов"](https://docs.astro.build/en/concepts/islands/)
+:::
+```
```
Comment (delucis) [inline docs/src/content/docs/ru/guides/authoring-content.md:144]:
````suggestion
:::tip[Знали ли вы?]
Astro позволяет создавать быстрые сайты с помощью ["архитектуры островов"](https://docs.astro.build/ru/concepts/islands/)
:::

```md
:::tip[Знали ли вы?]
Astro позволяет создавать быстрые сайты с помощью ["архитектуры островов"](https://docs.astro.build/ru/concepts/islands/)
:::
```
````

File: docs/src/content/docs/ru/manual-setup.mdx [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+как показано во [введении](/ru/getting-started/#creating-a-new-project).
+Если вы хотите добавить Starlight к существующему проекту Astro, это руководство расскажет, как это сделать.
+
+## Настройка Starlight
+
+Для работы по этому руководству вам понадобится существующий проект на Astro.
+
+### Интеграция Starlight
+
+Starlight является [интеграцией Astro](https://docs.astro.build/en/guides/integrations-guide/).
```
Comment (delucis) [inline docs/src/content/docs/ru/manual-setup.mdx:18]:
```suggestion
Starlight является [интеграцией Astro](https://docs.astro.build/ru/guides/integrations-guide/).
```

File: docs/src/content/docs/ru/manual-setup.mdx [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+		}),
+	],
+});
+```
+
+Найдите все доступные параметры в [справочнике конфигурации Starlight](/ru/reference/configuration/).
+
+### Настройка коллекций контента
+
+Starlight создан на основе [коллекций контента](https://docs.astro.build/en/guides/content-collections/) Astro,
```
Comment (delucis) [inline docs/src/content/docs/ru/manual-setup.mdx:67]:
```suggestion
Starlight создан на основе [коллекций контента](https://docs.astro.build/ru/guides/content-collections/) Astro,
```

File: docs/src/content/docs/ru/manual-setup.mdx [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+- astro.config.mjs
+
+</FileTree>
+
+В будущем мы планируем улучшить такой вариант использования путей,
+чтобы избежать необходимости в лишнем вложенном каталоге в `src/content/docs/`.
+
+### Использование Starlight с SSR
+
+На данный момент Starlight не поддерживает [развертывание с SSR](https://docs.astro.build/en/guides/server-side-rendering/)
```
Comment (delucis) [inline docs/src/content/docs/ru/manual-setup.mdx:135]:
```suggestion
На данный момент Starlight не поддерживает [развертывание с SSR](https://docs.astro.build/ru/guides/server-side-rendering/)
```

**Review by delucis — COMMENTED** [review]
Oh, actually, just realised one more thing to check: links to `docs.astro.build` can also use the `/ru/` link instead of `/en/`. Will update these before merging.

**Pull request merged**